client/checkout/woopay/express-button/woopay-express-checkout-payment-method.js

```diff
@@ -1,3 +1,9 @@
+/**
+ * External dependencies
+ */
+import { useCallback } from 'react';
```

Comment:
I was a little concerned about these imports, but I see there wasn't any file size or download size different on the front-end. Just wanted to confirm if there's any other performance aspect to be concerned with these?

Reply:
Nope. Since they're React stuff, they're already included in the bundle so no concerns on that aspect.